src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import Tooltip from 'rc-tooltip';
+import DefaultTooltip from 'shared/components/DefaultTooltip';
```

Comment:
oh, it worked for me. I assumed it was going from the project root directory, I didn't realize it was an alias. Should I change it?